src/Flags.cpp

```diff
@@ -50,6 +54,13 @@ void registerFastFlags(std::unordered_map<std::string, std::string>& fastFlags,
     {
         onWarning(std::string("Unknown FFlag: ") + key);
     }
+
+    // These are required flags for internal functionality.
```

Comment:
We shouldn't force these enabled, since the point of FFlags here are to dynamically opt into the configuration (and have the ability to opt out if there are problems)

Reply:
The documentation system actually can crash with these off and it fails like 7 tests. I would have to enable the fflags in each failing fixture as the alternative.

Reply:
Kept the flags forced in the testing suite as the alternative